osx/Fuse/build.sh

```diff
@@ -18,30 +18,26 @@ git clone --recursive -b osxfuse-$version git://github.com/osxfuse/osxfuse.git o
 rm -rf /tmp/kbfuse*
 cd osxfuse
 # If you get an error compiling you might have to run `brew link gettext --force` (see https://github.com/osxfuse/osxfuse/issues/149).
-./build.sh -t fsbundle
+# build for 10.11, and have osxfuse builder symlink other versions.
+./build.sh -v 5 -t fsbundle -- -s 10.11 -d 10.11 --kext=10.11 --kext="10.12->10.11" --kext="10.13->10.11" --kext="10.14->10.11"
 
 cd $dir
 rm -rf kbfuse.bundle
 ditto /tmp/kbfuse/fsbundle/kbfuse.fs kbfuse.bundle
 
-# Fix sym links for supported OS's
-cd $dir/kbfuse.bundle/Contents/Extensions
-ln -s 10.10 10.11
-ln -s 10.10 10.12
```

Comment:
Removed because `build.sh` with those flags already does this sort of linking automatically.